packaging/deb-systemd/debian/changelog

```diff
@@ -0,0 +1,5 @@
+mackerel-agent (0.41.1-1.systemd) stable; urgency=low
```

Comment:
deb package filename is determined by this name and version, so this version postfix `.systemd` is necessary not to create deb file with same filename as normal deb package.

`releng` is modified at https://github.com/mackerelio/mackerel-agent/pull/360/commits/ff8152d1734bb68b42fb66d8ec908826c89b3bdb so that newly created changelog will have `.systemd` postfix in debian-version.